websocket.js.org

YShenZe opened the pull request:
<!--

Thanks for creating a pull request to request a new subdomain from JS.ORG

Before creating your pull request, please complete the following steps:

- Ensure that your pull request changes only the cnames_active.js file, adding a single new line for your subdomain request
- Tick the two checkboxes, agreeing to the sentences, below by placing an x inside the square brackets ([ ] becomes [x])
- Add a link (GitHub repository, Vercel deployment, etc.) and explanation below for your content so we can validate your request

-->

- [ ✓] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [ ✓] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)
- The site content can be seen at <link>

> The site content is ...

MattIPv4 commented on cnames_active.js:
```diff
@@ -3700,7 +3700,8 @@ var cnames_active = {
   "zyx": "zyx.alwaysdata.net",
-  "zyy": "zyyou.github.io/notes"
+  "zyy": "zyyou.github.io/notes",
+  "websocket": "yshenze.github.io"
```
Why is this hosted at your personal GitHub Pages URL, rather than being in a repository specifically for the tester (which would then be at yshenze.github.io/whatever)?

Also, please see and resolve the CI error, your entry needs to be in alphabetical order, not dumped at the end of the file.

indus commented:
@YShenZe besides the problem with the alphabetical order, I would like to ask you to request a subdomain like `ws-debug` or `websocket_test` or similar as the `websocket` is quite generic for a tool that is quite specific in its functionality.

And as a result of using a user repo for your project instead of a project-repo there is a name-mismatch.
FYI: https://github.com/js-org/js.org/wiki/Subdomain-Determination

YShenZe commented:
Okay, thanks for reminding me.

YShenZe commented:
And I always thought that Github Pages could only apply for one, but I found out that I could apply for more than one.

YShenZe commented:
It has been revised according to the requirements.Forgive me for not knowing how to modify the pr title.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```ws-debug.js.org``` to make it work.
I'll merge after we have "contact". Please keep this PR open.

YShenZe commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```ws-debug.js.org``` to make it work.
> I'll merge after we have "contact". Please keep this PR open.

I've added the CNAME file.

indus commented:
Welcome https://ws-debug.js.org

indus commented:
You have to open a new PR and add the `// noCF` comment behind your line. FYI: https://github.com/js-org/js.org/blob/master/cnames_active.js#L17-L20

YShenZe commented:
A new Pr has been opened.
